lib/cylc/tests/test_subprocpool.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python3
```

Comment:
I know we might remove these (#3041), but as I spotted the old value on `master`, decided to change to python3 for now.